elixir/apps/web/lib/web/mailer/auth_email.ex

```diff
@@ -59,4 +59,22 @@ defmodule Web.Mailer.AuthEmail do
       remote_ip: "#{:inet.ntoa(remote_ip)}"
     )
   end
+
+  def new_user_email(
```

Comment:
Can we also add a button on the actor show page to send it? I feel that would be really valuable and super easy to implement. And then we can name this function something like "setup_instructions_email" or "invitation_email".

Reply:
@bmanifold we can do it there or on the individual actors page - up to you, only thing is that I've removed dropdowns because they were horrible on mobile and it was always just one button behind them